cmd/instances.go

```diff
@@ -172,6 +173,7 @@ be used as the error message.
 			DomainAliases: flagDomainAliases,
 			Locale:        flagLocale,
 			UUID:          flagUUID,
+			OIDCID:        flagOIDCID,
```

Comment:
(Nitpick) Wouldn't be better to have agnostic naming only related to oauth and not to only oidc, to allow more generic usage like Github/France Connect/whoever other oauth provider connection?

Reply:
Maybe, but for the moment, I appreciate that we talk of OIDC for authentication and of OAuth for konnectors (using OAuth for both will be confusing, even if it is something that we can't avoid on the long term)

Reply:
OK :)